capi/geos_c.h.in

```diff
@@ -754,6 +754,24 @@ extern void GEOS_DLL GEOSGeom_destroy_r(
     GEOSContextHandle_t handle,
     GEOSGeometry* g);
 
+/* ========= Coverages ========= */
+
+/** \see GEOSCoverageIsValid */
+int
+GEOSCoverageIsValid_r(
+    GEOSContextHandle_t extHandle,
+    const GEOSGeometry* input,
+    double gapWidth,
+    GEOSGeometry** output);
+
+/** \see GEOSCoverageSimplify */
+extern GEOSGeometry GEOS_DLL *
+GEOSCoverageSimplifyVW_r(
+    GEOSContextHandle_t extHandle,
+    const GEOSGeometry* input,
+    double tolerance,
+    int preserveBoundary);
```

Comment:
In a future version of this, it would be nice to specify that small polygons are allowed to collapse. Would it make sense to rename to the last param to `flags` to accommodate this and other future additions? Leaving it as-is doesn't preclude renaming it later, anyway, as we did with `GEOSVoronoiDiagram`